internal/api/v1/router.go

```diff
@@ -10,16 +10,28 @@ import (
 func Router() *httprouter.Router {
 	router := httprouter.New()
 	router.HandlerFunc("GET", "/api/v1/info", InfoController{}.Info)
+
+	// List, show, and delete applications
 	router.HandlerFunc("GET", "/api/v1/orgs/:org/applications", ApplicationsController{}.Index)
 	router.HandlerFunc("GET", "/api/v1/orgs/:org/applications/:app", ApplicationsController{}.Show)
 	router.HandlerFunc("DELETE", "/api/v1/orgs/:org/applications/:app", ApplicationsController{}.Delete)
 
+	// Bind and unbind services to/from applications
+	router.HandlerFunc("POST", "/api/v1/orgs/:org/applications/:app/services", ApplicationsController{}.Bind)
```

Comment:
I'm not sure but, strictly speaking, the resource we are creating is neither a Service not an Application, it's the thing in between (ServiceBinding?) Thus the url should be something like:

```
POST /api/v1/orgs/:org/applications/:app/servicebindings
```
the controller would be `ServiceBindingsController` and the action would be `Create`. The end result is the same but we keep the API resource-based and "semantic". A POST to an endpoint that ends with `services` should generally create a Service resource.
What do you think?

Reply:
While I can see changing the name to `service bindings`, I am less source about using a separate controller.
I am modeling it as a sub-collection resource in the application
See https://restfulapi.net/resource-naming .
In this sense I am creating an element in the sub-collection of the application resource.
Or maybe I misunderstood that page.

@manno Can you weigh in ?